DeleteCFStack/index.js

```diff
@@ -1,64 +1,75 @@
 const AWS = require('aws-sdk');
 const cf = new AWS.CloudFormation();
 
-async function handleWebhookEvent(webhookBody) {
-    const action = webhookBody['action'];
-    const pullRequest = webhookBody['pull_request'];
-    let message = 'Event ignored';
+function parsewebhookBody(eventType, webhookBody) {
+  let shouldDeleteStack, branchName, repoName;
 
-    // If no pull_request, ignore event
-    if(!pullRequest) {
-        return { statusCode: 200, message: message };
-    }
+  if (eventType === 'pull_request') {
+    branchName = pullRequest['head']['ref'];
+    repoName = pullRequest['head']['repo']['name'];
```

Comment:
Did you mean to update `pullRequest` here to `webhookBody['pull_request']`?

Reply:
DUDE - yes - GOOD EYES. Updating now.